Source/SwiftLintFramework/Documentation/RuleListDocumentation.swift

```diff
@@ -33,8 +33,9 @@ public struct RuleListDocumentation {
     // MARK: - Private
 
     private var indexContents: String {
-        let defaultRuleDocumentations = ruleDocumentations.filter { !$0.isOptInRule }
-        let optInRuleDocumentations = ruleDocumentations.filter { $0.isOptInRule }
+        let defaultRuleDocumentations = ruleDocumentations.filter { !$0.isOptInRule && !$0.isAnalyzerRule }
```

Comment:
```suggestion
        let defaultRuleDocumentations = ruleDocumentations.filter { !$0.isOptInRule }
```
Analyzer rules are opt-in by default.